### ISSUE TITLE
Map rotation

### ISSUE THREAD

**Issue by angoca** [fieldpapers/fieldpapers#271]
It will be very interesting to rotate the background map. It allows to adjust the streets or other elements according to the sheets.

Not every single object is oriented north south, and for this reason, a mapping party is not easy to assign streets in a grid base dsitribution city, as Bogota, Barcelona, or others.

Sometimes, the atlas have corners out of the scope of the mapping party.

**Comment by almccon** [comment]
@angoca Good idea! I think #179 is a duplicate. Can you confirm that #179 describes your request?

**Comment by angoca** [comment]
Yes, this could be a duplicate, but it is related only about rotation. The other issue describe more requested features.

**Comment by openbrian** [comment]
+1, I have definitely needed this feature for a few mapping events.

**Comment by quincylvania** [comment]
As of 2023, Leaflet doesn't support map rotation and [doesn't plan to](https://github.com/Leaflet/Leaflet/issues/268#issuecomment-421838981). This makes things trickier.

Note also that iD doesn't currently support rotation, so any rotated fieldpapers atlases would look skewed in that editor and any written notes might appear upside down.

**Comment by angoca** [comment]
What about rotating the sheets on top of the map? The result is the same.

**Comment by angoca** [comment]
<img width="1201" alt="Screen Shot 2023-04-26 at 12 25 26 PM" src="https://user-images.githubusercontent.com/294317/234655308-0ab883bc-60fc-4316-ae4b-3f0dce2ede09.png">
Rotating the sheets, something about this.
